example/src/app/service/hero.service.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { inject, TestBed } from '@angular/core/testing'
+import { HttpClient, HttpClientModule, HttpRequest } from '@angular/common/http'
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing'
+import { heroesUrl, HeroService } from './hero.service'
+
+describe('Service: GoogleBooks', () => {
+  let service: HeroService
+  let backend: HttpTestingController
+
+  const expectedData = {
+    id: 1,
+    name: 'Test hero'
+  }
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientModule,
+        HttpClientTestingModule,
+      ],
+      providers: [
+        HeroService,
+      ],
+    })
+
+    spyOn(console, 'warn')
+    spyOn(console, 'error')
```

Comment:
can we just `jest.spyOn`? `global.spyOn` comes from Jasmine

Reply:
oh ya we can replace that by jest.spyOn